Dockerfile.eas-admin

```diff
@@ -8,9 +8,8 @@ ENV ADMIN_DIR=/eas/emergency-alerts-admin
 COPY . $ADMIN_DIR
 
 # Run emergency-alerts-admin
-RUN $PYTHON_VERSION -m venv $VENV_ADMIN
-RUN . $VENV_ADMIN/bin/activate && pip3 install wheel
-RUN cd $ADMIN_DIR && . $VENV_ADMIN/bin/activate && make bootstrap
+RUN $PYTHON_VERSION -m venv $VENV_ADMIN && . $VENV_ADMIN/bin/activate && \
```

Comment:
This is fine to do in one RUN command but it will mean that it won't be able to cache the other layers (virtual env creation on pip installing etc). I think it may make the image slightly smaller though so its personal choice.